packages/apollo/__tests__/getCollection.spec.ts

```diff
@@ -1,20 +1,11 @@
 import getCollection from '../src/api/getCollection'
+import { createMockContext } from '../__mocks__/mockContext'
 
-const queryMock = jest.fn()
 
-const mockClient = { query: queryMock}
-
-const mockExtendQuery = jest.fn();
-
-const mockContext: any = {
-  extendQuery: mockExtendQuery,
-  client: {
-    apolloClient: mockClient
-  }
-}
 
 describe('[shopify-apollo] mapping of params into the qraphql client', () => {
-  it('should map params the the collection query', async () => {
+  it('should map params the collection query', async () => {
+    const mockContext = createMockContext()
```

Comment:
you can make desctructurization here
```suggestion
    const { context: mockContext, extendQuery: mockExtendQuery } = createMockContext()
```